intranet/static/js/files.js

```diff
@@ -1,91 +1,96 @@
 /* global $ */
-$(document).ready(function() {
+$(function() {
     // check for drop support, file support, and file upload support
     if (Modernizr.draganddrop && window.File && window.FormData) {
         $("#additional-upload-option").addClass("visible");
         var endpoint = $("#directory-list").attr("data-endpoint");
         // dragover event must be cancelled to drop
-        $(window).on("dragover", function(e) {
-            e.originalEvent.dataTransfer.dropEffect = "copy";
-            e.preventDefault();
-        });
-        $(window).on("dragenter", function(e) {
-            var flag = false;
-            var dt = e.originalEvent.dataTransfer;
-            if (dt.types) {
-                for (var i = 0; i < dt.types.length; i++) {
-                    if (dt.types[i] == "Files") {
-                        flag = true;
-                        break;
+        $(window).on({
+            "dragover": function(e) {
+                e.originalEvent.dataTransfer.dropEffect = "copy";
+                e.preventDefault();
+            },
+            "dragenter": function(e) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Function 'undefined' has a complexity of 5.](https://www.codacy.com/app/tjhsst-ion/ion/file/2769576146/issues/source?bid=3212528&fileBranchId=3355681#l13)